testsuite/MDAnalysisTests/analysis/test_dssp.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import glob
+import MDAnalysis as mda
+
+from MDAnalysis.analysis.dssp import DSSP, translate
+from MDAnalysisTests.datafiles import DSSP as DSSP_FOLDER
+from MDAnalysisTests.datafiles import TPR, XTC
+
+
+@pytest.mark.parametrize("pdb_filename", glob.glob(f"{DSSP_FOLDER}/?????.pdb.gz"))
```

Comment:
Would just doing `*.pdb.gz` here be better? At least if someone somehow adds an extra entry with a different name it'll get picked up?

Reply:
I agree.

Or are you using it to only load "good" PDBs and exclude "1mr1D_failing.pdb.gz" ?

If that's the case, add comment here and a README to the DSSP_FOLDER where you state the assumption that anything matching the glob pattern `?????.pdb.gz` is a correct input file and will be tested automatically.

Reply:
Yes, that's exactly the reason.
I added a README and a line to the file.